cognite/extractorutils/util.py

```diff
@@ -501,3 +502,7 @@ def handle_cognite_errors(exception: CogniteException) -> bool:
         return True
 
     return {CogniteException: handle_cognite_errors}
+
+
+def _timestamp_to_datetime(ts: int) -> datetime:
```

Comment:
I can imagine this being useful in extractors as well, maybe just make it public?

While we're at it, should we add the inverse as well?